src/peergos/server/util/JavaPoster.java

```diff
@@ -52,6 +52,7 @@ private CompletableFuture<byte[]> post(String url, byte[] payload, boolean unzip
             conn = (HttpURLConnection) buildURL(url).openConnection();
             if (timeoutMillis >= 0)
                 conn.setReadTimeout(timeoutMillis);
+            conn.setConnectTimeout(1_000);
```

Comment:
seems low. debugging code?

Reply:
1s is huge for connection latency. UK to Aus should be < .3s.